PencilCase/PencilCase/scripts/Pages/ProductEditor.ts

```diff
@@ -145,16 +145,16 @@ export class ProductEditor extends PageBase {
     }
 
     private reSetEditingFields = () => {
-        this.name(this.parameters.product.Name);
-        this.description(this.parameters.product.Description);
-        this.inventory(this.parameters.product.Inventory);
-        this.retailPrice(this.parameters.product.RetailPrice);
-        this.retailWholesalePrice(this.parameters.product.RetailWholesalePrice);
-        this.times(this.parameters.product.Times);
-        this.retailUnit(this.parameters.product.RetailUnit);
-        this.wholesaleUnit(this.parameters.product.WholesaleUnit);
-        this.wholesalePrice(this.parameters.product.WholesalePrice);
-        this.imageSource(this.parameters.product.Image);
+        this.name(this.originalProduct().Name);
```

Comment:
When saving, it will update this originalProduct instance, so always use this to initialize data when entering editing mode.